src/commands/init.js

```diff
@@ -0,0 +1,32 @@
+const {Command} = require("@oclif/command")
+const {cli} = require('cli-ux');
+const chalk = require('chalk');
+const tree = require('tree-node-cli');
+
+const Boilerplate = require('../prompts/boilerplate');
+const Build = require('../tasks/build');
+
+class Init extends Command {
+	// TODO: should we use args? (e.g. for component name instead of prompting)
```

Comment:
Prompting is great, let's keep the prompting. If we decide later we want arguments, we can add them later :-)
```suggestion
```